tests/functional/sync_reset_password.js

```diff
@@ -151,9 +152,10 @@ define([
         .end()
 
         .closeCurrentWindow()
+        .end()
+
         // switch to the original window
         .switchToWindow('')
-        .end();
```

Comment:
This seems to be where ESLint is crying. Need a ";" at the end of line 158, `.switchToWindow('');`, or somethin'.